pallets/registry/src/lib.rs

```diff
@@ -277,21 +278,14 @@ pub mod pallet {
 			registry_id: T::RegistryId,
 			token_id: T::TokenId,
 			asset_info: T::AssetInfo,
-			mint_info: MintInfo<T::Hash, T::Hash>,
+			mint_info: MintInfo<T::Hash, H256>,
```

Comment:
Same here. Lets use T::Hash

Reply:
Again, to prevent type mismatches
![image](https://user-images.githubusercontent.com/81358014/133805629-d6f4b567-b22a-4d5f-84fa-f0a458fa5b53.png)